nautobot/extras/jobs.py

```diff
@@ -410,6 +411,10 @@ def time_limit(cls):  # pylint: disable=no-self-argument
     def has_sensitive_variables(cls):  # pylint: disable=no-self-argument
         return getattr(cls.Meta, "has_sensitive_variables", True)
 
+    @classproperty
+    def supports_dryrun(cls):  # pylint: disable=no-self-argument
+        return isinstance(getattr(cls, "dryrun", None), DryRunVar)
```

Comment:
We should probably throw a warning on import if `dryrun` is not an instance of `DryRunVar`

Reply:
Tracking in #3087